src/discovery/azure/Akka.Discovery.Azure/Model/ClusterMember.cs

```diff
@@ -0,0 +1,130 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="ClusterMemberEntity.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using Azure;
+using Azure.Data.Tables;
+using Google.Protobuf;
+
+namespace Akka.Discovery.Azure.Model
+{
+    // If anything throws InvalidOperationException, then the data is corrupted
+    public class ClusterMember: IEquatable<ClusterMember>
+    {
+        internal const string PayloadName = "payload";
+        internal const string LastUpdateName = "lastUpdate";
+        
+        public ClusterMember(TableEntity entity)
+        {
+            PartitionKey = entity.PartitionKey;
+            RowKey = entity.RowKey;
+            Timestamp = entity.Timestamp;
+            ETag = entity.ETag;
+
+            Raw = entity;
+            Proto = ClusterMemberProto.Parser.ParseFrom(entity.GetBinary(PayloadName));
+            LastUpdate = new DateTime(entity.GetInt64(LastUpdateName).Value);
+        }
+        
+        #region Required fields
+        public string PartitionKey { get; }
+        public string RowKey { get; }
+        public DateTimeOffset? Timestamp { get; }
+        public ETag ETag { get; }
+        #endregion
+
+        internal TableEntity Raw { get; }
+        internal ClusterMemberProto Proto { get; }
+        
+        public string ServiceName => PartitionKey;
+        public string Host => Proto.Host;
+        public IPAddress Address => IPAddress.Parse(Proto.Address);
+        public int Port => Proto.Port;
+        public DateTime Created => Proto.Created.ToDateTime();
+        public DateTime LastUpdate { get; }
```

Comment:
As per changes in the spec, the cluster member table data model is stripped to the bare minimum. The only fields exposed are the ones needed by the query for TTL scheme support.
All the rest of the supporting data are packed inside a byte array using protobuf instead, for data versioning reasons